apps/full-site-editing/full-site-editing-plugin/starter-page-templates/class-starter-page-templates.php

```diff
@@ -148,12 +133,11 @@ public function enqueue_assets() {
 
 		);
 		$site_info = get_option( 'site_contact_info', array() );
-		$config    = array(
+		$config    = apply_filters( 'full_site_editing_starter_page_templates_config', array(
```

Comment:
Would it be good if the filter described the fact that this is data that is localised into JS?

Reply:
@obenland suggested to solve this by adding a doc comment for the filter. I didn't know they were a thing. Updating in a second